htdocs/comm/propal.php

```diff
@@ -2256,6 +2256,12 @@
 		$linktoelem = $form->showLinkToObjectBlock($object);
 		if ($linktoelem) print '<br>'.$linktoelem;
 
+        // Show links to link elements
```

Comment:
We should not need this. All links we want to see should be reported into showLinkToObjectBlock.
So if a special cas must be managed, it should be inside function